backend/consultations/emails.py

```
@@ -0,0 +1,46 @@
+        """
+
+    send_mail(
+        subject,                        # 제목
+        message,                        # 내용
+        settings.DEFAULT_FROM_EMAIL,    # 보내는 사람
+        [settings.ADMIN_EMAIL]          # 받는 사람
+    )
+
+def send_response_notification(consultation):
+    """ 답변 완료 시 신청자에게 알림 이메일 전송 """
+    subject = f'[세무법인 아람] 상담 답변이 등록되었습니다'
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)